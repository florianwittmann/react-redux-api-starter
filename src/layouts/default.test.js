import React from "react";
import { shallow } from "enzyme";
import DefaultLayout from "./default";
test("should test the layout comp without titles", () => {
  const wrapper = shallow(<DefaultLayout />);
  expect(wrapper).toMatchSnapshot();
});

test("should test the layout comp with title and subtitle", () => {
  const wrapper = shallow(
    <DefaultLayout title="Layout title" subtitle="Layout subtitle" />
  );
  expect(wrapper).toMatchSnapshot();
});
