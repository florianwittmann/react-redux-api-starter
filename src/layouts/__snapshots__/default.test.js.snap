// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the layout comp with title and subtitle 1`] = `
<div>
  <section
    className="hero is-primary"
  >
    <div
      className="hero-body"
    >
      <div
        className="container"
      >
        <h1
          className="title"
        >
          Layout title
        </h1>
        <h2
          className="subtitle"
        >
          Layout subtitle
        </h2>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    />
  </section>
</div>
`;

exports[`should test the layout comp without titles 1`] = `
<div>
  <section
    className="hero is-primary"
  >
    <div
      className="hero-body"
    >
      <div
        className="container"
      />
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    />
  </section>
</div>
`;
